### PR TITLE
Feat/225 226 move menu and logo to top

### LINKED ISSUE #226: Move menu up to the top
image

## Context
Arguably a more standard placement and fits better with the new visual hierarchy

### PR DESCRIPTION
**Pull request by braddf**
# Pull Request

## Description

Moved nav bar to top and logo into it from map.

Fixes #226 and #225 

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
I just checked out the branch, when I hover the menus no logger appear. Like under data, there should be text. 

This could be becasue the text goes upwards and then it cant be seen?

THis is on dev at the moment
Screenshot 2022-09-23 at 13 57 34

File: apps/nowcasting-app/components/logo.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const OCFlogo = () => (
+  <a
+    className="py-[4px] z-20"
+    href="https://www.openclimatefix.org/"
+    target="_blank"
+    rel="noreferrer"
+  >
+    <img src="/OCF_icon_wht.svg" alt="ofc" width={60} />
+  </a>
+);
```
Comment (rachel-labri-tipton) [inline apps/nowcasting-app/components/logo.tsx:10]: good idea to put this in its own file

Comment (rachel-labri-tipton) [reply apps/nowcasting-app/components/logo.tsx:10]: everything looks good to me except for the tooltip pop-ups that peter pointed out.

**Comment by braddf** [comment]
Thanks for the spot @peterdudfield and @rachel-labri-tipton - should be fixed and conflicts updated too 🤞

File: apps/nowcasting-app/components/charts/remix-line.tsx [hunk 2]
```diff
@@ -194,21 +194,24 @@ const RemixLine: React.FC<RemixLineProps> = ({
...
-                  <div className="p-2 bg-mapbox-black bg-opacity-70 shadow">
+                  <div className="px-3 py-2 bg-mapbox-black bg-opacity-70 shadow">
                     <ul className="">
                       {Object.entries(data)
                         .reverse()
                         .map(([name, value]) => {
                           if (name === "formatedDate") return null;
+                          const textClass = ["FORECAST", "PAST_FORECAST"].includes(name)
+                            ? "font-semibold"
+                            : "font-normal";
```
Comment (rachel-labri-tipton) [inline apps/nowcasting-app/components/charts/remix-line.tsx:205]: great to see how that works.

**Review by rachel-labri-tipton — APPROVED** [review]
these changes look good to me. i see you moved to the tooltip to the top.

**Pull request merged**